_rules/SC3-1-1-html-has-lang.md

```diff
@@ -0,0 +1,94 @@
+---
+name: Existence of HTML Lang attribute
+
+description: |
+  This rule checks that the `html` element has a [non-empty][] `lang` or `xml:lang` attribute.
+
+success_criterion:
+- 3.1.1
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Annika Nietzio
+- Jey Nandakumar
+---
+
+## Test Procedure
+
+### Applicability
+
+The root element of the page, if it is an `html` element.
+
+### Expectation
+
+The test target has a `lang` or `xml:lang` attribute.
```

Comment:
Add non-empty here like in the description: 
"The test target has a [non-empty][] `lang` or `xml:lang` attribute."

Reply:
Agreed.